inferno/extensions/layers/convolutional.py

```diff
@@ -1,77 +1,125 @@
 import torch.nn as nn
-from ..initializers import OrthogonalWeightsZeroBias, KaimingNormalWeightsZeroBias, \
-    SELUWeightsZeroBias
+import sys
+import functools
+from ..initializers import (
+    OrthogonalWeightsZeroBias,
+    KaimingNormalWeightsZeroBias,
+    SELUWeightsZeroBias,
+)
 from ..initializers import Initializer
 from .activations import SELU
 from ...utils.exceptions import assert_, ShapeError
-
-
-__all__ = ['ConvActivation',
-           'ConvELU2D', 'ConvELU3D',
-           'ConvSigmoid2D', 'ConvSigmoid3D',
-           'DeconvELU2D', 'DeconvELU3D',
-           'StridedConvELU2D', 'StridedConvELU3D',
-           'DilatedConvELU2D', 'DilatedConvELU3D',
-           'Conv2D', 'Conv3D',
-           'BNReLUConv2D', 'BNReLUConv3D',
-           'BNReLUDepthwiseConv2D',
-           'ConvSELU2D', 'ConvSELU3D',
-           'ConvReLU2D', 'ConvReLU3D',
-           'BNReLUDilatedConv2D', 'DilatedConv2D',
-           'GlobalConv2D']
+from ...utils.partial_cls import register_partial_cls
+
+__all__ = [
```

Comment:
Could we autogenerate this as well, by appending the names to `__all__` in `_register_conv_cls`?

Reply:
yes, we could append to `__all__` or just remove the `__all__` in which case python basicaly generates this for us.